Header header
mavros_msgs/Waypoint departure
mavros_msgs/Waypoint arrival
mavros_msgs/Waypoint[] path
